devtools/platform-descriptor-json/src/main/resources/codestarts/quarkus/core/buildtool/maven/codestart.yml

```diff
@@ -14,7 +14,6 @@ language:
         version: 3.8.1
       maven-surefire-plugin:
         version: 2.22.1
```

Comment:
But why do we still have the old versions there?

Reply:
@gsmet they are just default, I didn't start from your PR. But with this fix, the default will be overrided by the data from the `quarkus.properties`.

Reply:
OK, then, let's merge both PRs.